src/core/meshcop/joiner.cpp

```diff
@@ -67,8 +67,6 @@ Joiner::Joiner(ThreadNetif &aNetif):
     mState(kStateIdle),
     mCallback(NULL),
     mContext(NULL),
```

Comment:
`mCcitt` and `mAnsi` are only used after `Start()`. And what meaningful initial values for them?